ecommerce-ios/Views/ColorCell/ColorCell.swift

```diff
@@ -23,6 +23,7 @@ struct ColorCell: View {
                         .font(.system(size: 13.0))
                 }
             }
+            .contentShape(Rectangle())
```

Comment:
I would say another way to do this view is
```
var body: some View {
VStack(spacing: 8.0) {
            Circle()
                .foregroundColor(Color(hexString: viewModel.colorCode))
                .frame(width: 24.0, height: 24.0)
            Text(viewModel.name.capitalized)
                .foregroundColor(.charadeGray)
                .font(.system(size: 13.0))
        }
.frame(maxWidth: .infinity, idealHeight: 72.0)
}
```

But I'm not sure what's the benefit compares to .overlay

Reply:
It is my workaround solution. 

So without `.contentShape(Rectangle())`, the `ColorView` is only tappable when I tap on the circle and text, the spacing around is not tappable.

Basically, the `.contentShape(Rectangle())` makes the whole view tappable, and it resolves my problem 😅

I will give your solution a try and let you know if it works on swiftui channel 👍